### PR TITLE
Update databases.md

### PR DESCRIPTION
**Pull request by bigpreshy**
### What does it do?
Without removal of these first string, it bring some error messages

### Why is it needed?
This is will help new strapi users..

###  Describe the issue you are solving.
'replicaSet' is not recognized as an internal or external command,
operable program or batch file.
'authSource' is not recognized as an internal or external command,
operable program or batch file.
'retryWrites' is not recognized as an internal or external command,
operable program or batch file.
'w' is not recognized as an internal or external command,
operable program or batch file.

and the build fails.

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request

### CODE REVIEW

**Comment by bigpreshy** [comment]
Without removal of these first string, it bring some error messages which makes it fails to build

File: docs/v3.x/guides/databases.md [hunk 1]
```diff
@@ -312,6 +312,15 @@ MongoDB Atlas automatically exposes the database credentials into a single envir
 
 `mongodb://paulbocuse:<password>@strapi-heroku-shard-00-00-oxxxo.mongodb.net:27017,strapi-heroku-shard-00-01-oxxxo.mongodb.net:27017,strapi-heroku-shard-00-02-oxxxo.mongodb.net:27017/test?ssl=true&replicaSet=Strapi-Heroku-shard-0&authSource=admin&retryWrites=true&w=majority`
 
+
+You **must** use remove some string and it has to look like so:
```
[the remark this answers is not in the document]
Comment (bigpreshy) [reply docs/v3.x/guides/databases.md:316]: Ok

[the remark this answers is not in the document]
Comment (meganelacheny) [reply docs/v3.x/guides/databases.md:316]: Hello 👋 
Yes @derrickmehaffy I agree with the rephrasing you suggested. However, no need to put "must" in bold.

[the remark this answers is not in the document]
Comment (bigpreshy) [reply docs/v3.x/guides/databases.md:316]: Ok...
I also have found a way to connect to mongo dB atlas without having to have mongo dB installed locally first as suggested by the documentation...

I am working on an article for this...

[the remark this answers is not in the document]
Comment (derrickmehaffy) [reply docs/v3.x/guides/databases.md:316]: Do you want me to hold this PR then @bigpreshy ?

[the remark this answers is not in the document]
Comment (bigpreshy) [reply docs/v3.x/guides/databases.md:316]: No, you can go ahead... The article will different from this issue. Just that you one won't encounter this issue when deploying

[the remark this answers is not in the document]
Comment (derrickmehaffy) [reply docs/v3.x/guides/databases.md:316]: Can you make the change that @meganelacheny suggested and remove the bold around `must`

**Review by derrickmehaffy — CHANGES_REQUESTED** [review]

File: docs/v3.x/guides/databases.md (hunk 1 above)
Comment (derrickmehaffy) [inline docs/v3.x/guides/databases.md:316]: Can you remove the `**` around `must` please then I can merge this in

**Review by derrickmehaffy — CHANGES_REQUESTED** [review]

**Review by derrickmehaffy — APPROVED** [review]
LGTM

**Pull request merged**